src/xma/src/xmaapi/xmaadmin.cpp

```diff
@@ -247,16 +248,54 @@ xma_admin_session_destroy(XmaAdminSession *session)
     /*
     delete (XmaHwSessionPrivate*)session->base.hw_session.private_do_not_use;
     */
+    XmaHwSessionPrivate *priv1 = (XmaHwSessionPrivate*)session->base.hw_session.private_do_not_use;
+    if(priv1 != nullptr) {
```

Comment:
as per XRT repo coding style please use following for if and for loops
if () {
}

for () {
}

change it in rest of files also